docs/helm-chart/keda.rst

```diff
@@ -54,19 +50,22 @@ to set ``worker.persistence.enabled`` to ``false``)
        --namespace airflow \
        --set executor=CeleryExecutor \
        --set workers.keda.enabled=true \
-       --set workers.persistence.enabled=false
+
+A ``ScaledObject`` and an ``hpa`` will be created in the airflow namespace.
 
 KEDA will derive the desired number of celery workers by querying
 Airflow metadata database:
 
-.. code-block:: none
+.. code-block:: sql
 
```

Comment:
this does not work because it is not valid SQL. It is mix of Jinja and SQL.

Reply:
darn.  yes will fix that to.

Reply:
pushed.

Reply:
We could set it to `:: jinja` I think, but that probably expects it to be template to output HTML, so would likely be worse than no highlighting.

Reply:
Yup, tested it (on terminal with `pygmentize -l jinja`) and it looks ugly.